integration/tests/access/access_api_test.go

```diff
@@ -0,0 +1,295 @@
+package access
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/onflow/cadence"
+
+	sdk "github.com/onflow/flow-go-sdk"
+	client "github.com/onflow/flow-go-sdk/access/grpc"
+
+	"github.com/onflow/flow-go/engine/access/rpc/backend"
+	"github.com/onflow/flow-go/integration/testnet"
+	"github.com/onflow/flow-go/integration/tests/lib"
+	"github.com/onflow/flow-go/integration/tests/mvp"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// This is a collection of tests that validate various Access API endpoints work as expected.
+
+const (
+	simpleScript = `pub fun main(): Int { return %d; }`
```

Comment:
nit: Is the template needed? Maybe just:
```
simpleScript = `pub fun main(): Int { return 42 }`
simpleScriptVal = cadence.NewInt(42)
```

Reply:
addressed in [c8da7ec](https://github.com/onflow/flow-go/pull/4791/commits/c8da7ecca3c6db68ec81ef4e4b37a5de39c7294d)